geth/chain.py

```diff
@@ -112,7 +111,10 @@ def write_genesis_file(
 
     if config is None:
         config = {
+            "ethash": {},
             "homesteadBlock": 0,
+            "daoForkBlock": 0,
+            "daoForSupport": True,
```

Comment:
looks like this was supposed to be `daoForkSupport`  per the `v0.8.0` release notes? But if it hasn't caused any issues since then, :shrug:

Reply:
Good catch, I just re-organized it but looks like it has always said `daoForSupport`. I will change it to be correct though it may just be ignored since it hasn't caused issues.